traits.yml

```diff
@@ -96,36 +82,27 @@ specifications:
           - package: openassetio-mediacreation
             namespace: identity
             name: DisplayName
-          - package: openassetio-mediacreation
-            namespace: managementPolicy
-            name: Managed
-          - package: openassetio-mediacreation
-            namespace: managementPolicy
-            name: ResolvesFutureEntities
```

Comment:
These shouldn't be part of a specification for an entity, they are used when a Manager tells a host whether or not it wishes to be involved in read/publishing for this type of entity. Ie, they are set [here](https://github.com/gfilicetti/metadata-interchange/blob/7abf221748dacb1688e462331a36c1e213b516ae/plugin/gcpsample_asset_manager/GCPSampleAssetManagerInterface.py#L103) in [`managementPolicy`](https://openassetio.github.io/OpenAssetIO/classopenassetio_1_1v1_1_1manager_api_1_1_manager_interface.html#a7cfabfdd697739e7480d8df603b758dd).

Reply:
Don't I need them in there to be able to imbue them to a policy?

In my manager's managementPolicy method I imbue the policy with both of them so that the publishing.py code's if statements are satisfied. (but of course publishing.py comes from the Examples section so might be out of date?)

Reply:
The specifications form the "type" of entity. The host passes that to the manager to ask "what do you do with this kind of asset"? The manager then returns a `TraitsData` that has the traits that describe how it wishes to interact with them. Those traits are describing "what the manager is capable of", not "what kind of entity it is".

Reply:
The if statements are meant to be conditional based on what the manager is capable of, vs a requirement.